### PR TITLE
Updated bio information for workshop - Nick Strayer

### PR DESCRIPTION
**Pull request by nstrayer**
Hi Garrick, 

Sorry for my slow work on this. I tweaked a few things but you did a great job already. Sorry you had to take the time to do that!

### CODE REVIEW

File: content/authors/nick/_index.md [hunk 1]
```diff
@@ -18,14 +18,22 @@ projects:
   url: https://github.com/nstrayer/shinysense
   icon_pack: fab
   icon: fa-r-project
+- name: LiveFreeOrDichotomize
+  url: https://livefreeordichotomize.com/
+  icon_pack: fab
+  icon: fa-r-project
+- name: sbmR
+  url: https://github.com/tbilab/sbmR
+  icon_pack: fab
```
Comment (gadenbuie) [inline content/authors/nick/_index.md]:
```suggestion
  icon_pack: fas
```

**Comment by gadenbuie** [comment]
Thanks Nick! I suggested a small change, I think you want the `fas` icon pack for the paragraph icon.

**Comment by nstrayer** [comment]
Hi! Yes I do. I never got it to compile before making the pr but fixed the issue now (it was two full file paths). Commits to fix coming in two seconds.

**Comment by gadenbuie** [comment]
Oh good catch, thanks! I'm a big `here` fan it's hard to believe I missed that!

**Pull request merged**